docker/cyclus-deps/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:8.5
+FROM ubuntu:18.04 
```

Comment:
It seems to me that the only change one should have to make to solve #1504 should be to change 8.5 to 9 in this line and rebuild the cyclus-deps dockerhub image (why do we push this image manually anyway?). I'm concerned about this choice to use ubuntu rather than the more fundamental debian. Why go with ubuntu here?